hook/suite_test.go

```diff
@@ -35,16 +35,15 @@ var k8sClient client.Client
 var testEnv *envtest.Environment
 var testCtx, testCancel = context.WithCancel(context.Background())
 
-const (
+var (
```

Comment:
Can we use const instead?

Reply:
I see. This is a test code, so I think using var is acceptable.

Reply:
After thinking about this for a while, I changed my mind. `const` reduces the chance of programming errors and makes test more robust. It is more important than it looks. I've pushed revised version.